tests/ballerina-test/src/test/java/org/ballerinalang/test/functions/CustomFunctionTest.java

```diff
@@ -32,14 +32,14 @@
     public void testDuplicateFunction() {
         CompileResult compile = BCompileUtil.compile("test-src/functions/duplicate-function.bal");
         Assert.assertEquals(compile.getErrorCount(), 1);
-        BAssertUtil.validateError(compile, 0, "redeclared symbol 'foo'", 5, 1);
+        BAssertUtil.validateError(compile, 0, "redeclared symbol 'foobar'", 5, 1);
     }
 
     @Test(description = "Test defining ballerina function with duplicate parameters")
     public void testDuplicateParameters() {
         CompileResult compile = BCompileUtil.compile("test-src/functions/duplicate-parameters.bal");
         Assert.assertEquals(compile.getErrorCount(), 1);
-        BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 28);
+        BAssertUtil.validateError(compile, 0, "redeclared symbol 'param'", 1, 31);
```

Comment:
Can you explain what this 28 & 31 are?

Reply:
I changed the function name from `foo` to `foobar` in duplicate-function.bal file like following

```
function foobar(string param, int param) {
	string a;
	a = param;
}
```
You can seea duplicate args with `param` in 31 line.

Reply:
Can you put that as a comment in the test?

Reply:
Done.